var/spack/repos/builtin/packages/mpifileutils/package.py

```diff
@@ -65,11 +67,10 @@ def configure_args(self):
         else:
             args.append('--disable-lustre')
 
-        #  coming with v0.7
-        # if '+experimental' in self.spec:
-        #     args.append('--enable-experimental')
-        # else:
-        #     args.append('--disable-experimental')
+        if '+experimental' in self.spec:
+            args.append('--enable-experimental')
+        else:
+            args.append('--disable-experimental')
```

Comment:
I think you'll also want to check the version before you set these flags. For example, I doubt `--disable-experimental` will work for 0.6. The syntax to do this would be:
```python
if self.spec.satisfies('@0.7:'):
    if '+experimental' in self.spec:
       ...
```